docs/troubleshooting-faq/faq.md

```diff
@@ -0,0 +1,101 @@
+# Frequently asked questions
+
+### Why didn't the sample/project I specified when running the [data download script](../getting-started/accessing-resources/getting-access-to-data.md#using-the-download-data-script) download?
+
+First, we recommend using the `--dryrun` flag when running the `download-data.py` script to check which files _would_ be downloaded.
+This will confirm that there is nothing wrong with your internet connection and that you are properly [logged into your AWS profile](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+
+If running the script with `--dryrun` states that _only_ the `DATA_USAGE.md` file is being downloaded, this means the data files you are attempting to download do not exist.
+There are two main reasons why this might occur:
+
+1. Not all projects are available in `AnnData` format.
+If you are attempting to download `AnnData` format, please [consult the ScPCA documentation](https://scpca.readthedocs.io/en/stable/faq.html#which-samples-can-i-download-as-anndata-objects) to learn more about which types of projects do not have `AnnData` files.
+1. The sample or project ID(s) that you specified during download do(es) not exist.
+
+Therefore, you may wish to list some of the files in the data release S3 bucket to confirm the sample/project files you are trying to download actually exist.
+
+Data files in each release are organized on S3 as:
+```{ .console .no-copy title="Release file structure"}
+{Release}
+    ├── {Project ID}
+    │   └── {Sample ID}
+    │       └── {Library files}
+    ├── bulk_metadata.tsv (if applicable)
+    ├── bulk_quant.tsv (if applicable)
+    └── single_cell_metadata.tsv
+```
+
+To list all available files, you'll need to use the AWS CLI [in a terminal](../software-platforms/general-tools/using-the-terminal.md) and run `aws s3 ls` to list the contents of a given S3 bucket.
+Don't forget to [log into your AWS `openscpca` profile first](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session)!
+
+
+1. First, find the names of all the data releases, which are named based on their release date in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601).
+You can use the [`download-data.py`](../getting-started/accessing-resources/getting-access-to-data.md#download-data-structure) script with the `--list-releases` flag to do this:
```

Comment:
```suggestion

1. First, find the names of all the data releases, which are named based on their release date in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601).
You can use the [`download-data.py`](../getting-started/accessing-resources/getting-access-to-data.md#download-data-structure) script with the `--list-releases` flag to do this. 
Don't forget to [log into your AWS `openscpca` profile first](../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session)!
```
It seemed odd to talk about `aws s3 ls` when you don't show an example with it until after listing the releases.